docs/source/reference/pruners.rst

```diff
@@ -6,7 +6,8 @@ optuna.pruners
 The :mod:`~optuna.pruners` module defines a :class:`~optuna.pruners.BasePruner` class characterized by an abstract :meth:`~optuna.pruners.BasePruner.prune` method, which, for a given trial and its associated study, returns a boolean value representing whether the trial should be pruned. This determination is made based on stored intermediate values of the objective function, as previously reported for the trial using :meth:`optuna.trial.Trial.report`. The remaining classes in this module represent child classes, inheriting from :class:`~optuna.pruners.BasePruner`, which implement different pruning strategies.
 
 .. seealso::
-    :ref:`pruning` tutorial explains the concept of the pruner classes and a minimal example.
+    | :ref:`pruning` tutorial explains the concept of the pruner classes and a minimal example.
+    | :ref:`user_defined_pruner` tutorial could be helpful if you want to implement your own pruner classes.
```

Comment:
```suggestion
    :ref:`pruning` tutorial explains the concept of the pruner classes and a minimal example.
    
.. seealso::
    :ref:`user_defined_pruner` tutorial could be helpful if you want to implement your own pruner classes.
```

I prefer to divide independent blocks because these are different topics. I'd like to hear your impression.

Reply:
From the perspective of semantics, it would be better to separate them.
However, I thought that separating them would make the page a bit messy (see the first image), so I combined them into one (see the second image).

<img width="450" alt="Screen Shot 2022-02-27 at 14 14 25" src="https://user-images.githubusercontent.com/48990104/155869179-4b81fd42-1c8e-4deb-8967-b67474cf8e60.png">
<img width="450" alt="Screen Shot 2022-02-27 at 14 25 28" src="https://user-images.githubusercontent.com/48990104/155869446-5c093fd6-3e1e-4e95-a238-e63111ca2825.png">

Reply:
> I thought that separating them would make the page a bit messy

I think it is acceptable as it is seemingly more important to avoid confusion. Thanks for considering the user perspective.